Packs/Core/ReleaseNotes/3_0_97.md

```diff
@@ -0,0 +1,8 @@
+#### Integrations
+
+##### Investigation & Response
+
+- Updated the Docker image to: *demisto/google-cloud-storage:1.0.0.117186*.
+- Added two new optional arguments to the ***core-run-script-execute-commands*** command:
+  - ***is_raw_command***
+  - ***command_type***
```

Comment:
Release notes help users know what's new in the integration, write a message that explains _why_ we added these two args, or or what they do.

Reply:
Improved release notes in commit 8fece544d8feec9612c4915660d9d8e2dbb83fcd

Please let me know if additional improvements / edits are needed.